modules/apps/site/site-memberships-test/src/testIntegration/java/com/liferay/site/memberships/service/test/UserGroupRoleLocalServiceTest.java

```diff
@@ -74,8 +73,6 @@ public void testAddUserGroupRoles() throws Exception {
 		Assert.assertEquals(
 			userGroupRoles.toString(), 1, userGroupRoles.size());
 
-		EntityCacheUtil.clearLocalCache();
```

Comment:
@dantewang @yuhai 
The local cache is enabled or not depends on a property value. If you remove this line, and later we changed the property value for test environment, this test may fail.

To avoid unnecessary risk in future, let's keep the logic here. 

Thanks.
Tina.